.circleci/config.yml

```diff
@@ -502,16 +533,43 @@ workflows:
           <<: *on-integ-and-version-tags
       - platform_build:
           <<: *context
-          <<: *on-integ-and-version-tags
+          <<: *on-any-branch # revert on-integ-and-version-tags
           matrix:
             parameters:
               platform: [focal, bionic, xenial, centos7, ol8]
+              redis_version: ["6.0", "6.2", "7", "unstable"]
+            exclude:
+              - platform: bionic
+                redis_version: "6.0"
+              - platform: bionic
+                redis_version: "7"
+              - platform: bionic
+                redis_version: "unstable"
+              - platform: xenial
+                redis_version: "6.0"
+              - platform: xenial
+                redis_version: "7"
+              - platform: xenial
+                redis_version: "unstable"
+              - platform: centos7
+                redis_version: "6.0"
+              - platform: centos7
+                redis_version: "7"
+              - platform: centos7
+                redis_version: "unstable"
+              - platform: ol8
+                redis_version: "6.0"
+              - platform: ol8
+                redis_version: "7"
+              - platform: ol8
+                redis_version: "unstable"
```

Comment:
can't we just check the redis-server matrix for a single platform?

Reply:
yes but we want also to run the other platforms

Reply:
why? logic is the same for all of them.